cpp/src/io/json/json_column.cu

```diff
@@ -333,9 +319,18 @@ void make_device_json_column(device_span<SymbolT const> input,
                              rmm::mr::device_memory_resource* mr)
 {
   CUDF_FUNC_RANGE();
+  rmm::device_uvector<NodeIndexT> original_col_ids(col_ids.size(), stream);  // make a copy
+  thrust::copy(rmm::exec_policy(stream), col_ids.begin(), col_ids.end(), original_col_ids.begin());
+
+  // sort_by_key {col_id}, {row_offset} stable
```

Comment:
This comment sounds like it's sorting by two key columns (`col_id, row_offset`), but I only see one, `col_ids`. What does `{row_offset}` mean in this comment?

Reply:
`{col_id}` is key, `{row_offset}` is value